### ISSUE TITLE
Cesium visualization issue with asset id: 120215

### ISSUE THREAD

**Issue by osmondjia** [AnalyticalGraphicsInc/cesium-ion-community#338]
I have uploaded a geoJSON file in the Cesium ion, but there is a warning occurred that said "We're having trouble with your data. To help fix it, please open an issue in our Github repository."
What problems should be fixed?

**Comment by OmarShehata** [comment]
I think this is related to this issue: https://github.com/CesiumGS/cesium/issues/8042

If you change the arcType after adding your GeoJSON to the scene it should fix it:

```
var promise = Cesium.IonResource.fromAssetId(121192)
  .then(function (resource) {
    return Cesium.GeoJsonDataSource.load(resource);
  })
  .then(function (data) {
    for (var i = 0; i < data.entities.values.length; i++) {
        var entity = data.entities.values[i];
        if (Cesium.defined(entity.polyline)) {
            entity.polyline.arcType = Cesium.ArcType.GEODESIC;
        }
    }
    return viewer.dataSources.add(data);
  })
  .then(function (dataSource) {
    return viewer.zoomTo(dataSource);
  })
  .otherwise(function (error) {
    console.log(error);
  });
```

**Comment by lilleyse** [comment]
This was fixed in https://github.com/CesiumGS/cesium/pull/9028. The code works without the workaround now.

```
var promise = Cesium.IonResource.fromAssetId(121192)
  .then(function (resource) {
    return Cesium.GeoJsonDataSource.load(resource);
  })
  .then(function (data) {
    return viewer.dataSources.add(data);
  })
  .then(function (dataSource) {
    return viewer.zoomTo(dataSource);
  })
  .otherwise(function (error) {
    console.log(error);
  });
```

**Comment by lilleyse** [comment]
The fix will go out with the CesiumJS 1.72 release on August 3rd. It may also be deployed to ion before then.
